frontend/server/src/Controllers/QualityNomination.php

```diff
@@ -835,6 +828,45 @@ public static function apiResolve(\OmegaUp\Request $r): array {
         return ['status' => 'ok'];
     }
 
+    /**
+     * Updates a QualityNominations given an id and contents
+     *
+     * @param \OmegaUp\Request $r         The request.
+     *
+     * @omegaup-request-param string $contents
+     * @omegaup-request-param int $qualitynomination_id
+     *
+     * @return array{status: string}
+     */
+    public static function apiUpdate(\Omegaup\Request $r): array {
+        \OmegaUp\Controllers\Controller::ensureNotInLockdown();
+
+        // Validate request
+        $r->ensureMainUserIdentity();
+        self::validateMemberOfReviewerGroup($r);
+
+        $qualityNominationId = $r->ensureInt('qualitynomination_id');
+        $rawContents = $r->ensureString(
+            'contents',
+            fn (string $alias) => \OmegaUp\Validators::stringNonEmpty($alias),
```

Comment:
```suggestion
            fn (string $s) => \OmegaUp\Validators::stringNonEmpty($s),
```

nit: esto no es un alias.